contracts/SiphonReceiver.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.11;
+
+import { ERC20 } from "@openzeppelin/contracts-0.8/token/ERC20/ERC20.sol";
+import { Ownable } from "@openzeppelin/contracts-0.8/access/Ownable.sol";
+
+/**
+ * @title SiphonReciever
+ * @dev Takes rAURA deposits from rAURA on L1 and distributes them
+ *      When rewardClaimed is called on the Booster
+ */
+contract SiphonReceiver {
+    function queueRAura(uint256) external {
```

Comment:
Dont need to queue anything right? Just mint to here

Reply:
Naming weird

Reply:
Yeah we don't actually need this function, removed it 🖖